av/stream.pyi

```diff
@@ -22,5 +32,5 @@ class Stream:
     language: str | None
     type: Literal["video", "audio", "data", "subtitle", "attachment"]
 
-    def encode(self, frame=None): ...
-    def decode(self, packet=None): ...
+    def encode(self, frame: Frame | None = None) -> list[Packet]: ...
+    def decode(self, packet: Packet | None = None) -> list[Frame]: ...
```

Comment:
Make sure these methods overloaded in VideoStream and AudioStream respectively

Reply:
Yes they are, no worries